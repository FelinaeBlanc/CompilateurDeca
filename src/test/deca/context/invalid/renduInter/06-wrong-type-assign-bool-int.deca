{
boolean x = true;
int y = 4;
x = y;
}