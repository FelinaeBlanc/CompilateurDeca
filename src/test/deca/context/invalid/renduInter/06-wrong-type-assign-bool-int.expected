06-wrong-type-assign-bool-int.deca:4:4: Un type diff√©rent est attendu